src/styles/_class-overrides.scss

```diff
@@ -299,14 +299,16 @@
     //position languages at the bottom
     position: absolute;
     bottom: 0;
+    left: 0;
```

Comment:
What is the order for your properties here?

Reply:
There isn't one for now - I can implement one in my sass organization PR